frontend/src/components/SignUp/SignUpMobile.tsx

```diff
@@ -1,6 +1,201 @@
-import React from 'react';
+import { Box, Button, Grid, Link, TextField, Typography } from '@mui/material';
+import React, { FormEvent, useState } from 'react';
+import User from '../../types/User';
 import './signup.css';
 
-export default function SignUpMobile() {
-  return <div>Sign Up</div>;
+interface stateInterface {
```

Comment:
if this interface is the same on desktop and mobile, it can go in a common `SignUpBase.ts` file, along with any functions that have shared logic

Reply:
Forgot that the MUI elements actually resize themselves on mobile view, so there is no need for two different signup components. I've consolidated them.